src/paste3/model_selection.py

```diff
@@ -252,7 +163,7 @@ def plot_edge_curve(m_list, source_list, target_list):
     plt.show()
 
 
-def select_overlap_fraction_plotting(sliceA, sliceB, alpha=0.1, show_plot=True):
+def select_overlap_fraction(sliceA, sliceB, alpha=0.1, show_plot=True, numItermax=1000):
```

Comment:
Here and at several other places, `numItermax` (or `maxIter`) have been exposed (while keeping the original default values) so we can override them in tests.